integration/gui.test.js

```diff
@@ -0,0 +1,43 @@
+const puppeteer = require('puppeteer')
+const expect = require('expect-puppeteer')
+
+describe('Integration', () => {
```

Comment:
How do you feel about calling these `e2e`? I feel like `integration` is an overloaded superset.

Reply:
If you do a quick google for puppeteer testing it looks like `End to End` is a pretty common nomenclature e.g. https://egghead.io/courses/end-to-end-testing-with-google-s-puppeteer-and-jest